docs/en/docs/network/modules/multus-underlay/macvlan.md

```diff
@@ -1,41 +1,41 @@
-#Macvlan
+# Macvlan
 
 Macvlan is a network card virtualization solution for Linux, which can virtualize a physical network card into multiple virtual network cards.
-With Mutus, one or more Macvlan NICs can be assigned to Pods, so that Pods can communicate with the outside world through macvlan NICs.
+With Multus, one or more Macvlan network cards can be assigned to pods, so that pods can communicate externally through Macvlan network cards.
```

Comment:
```suggestion
With Multus, one or more Macvlan NICs can be assigned to pods, so that pods can communicate externally through Macvlan network cards.
```
NIC is preferred.